package.json

```diff
@@ -30,8 +30,7 @@
     "test": "ng test",
     "prebuild": "rimraf -- dist inline-template src/node_modules",
     "build": "gulp && ngc -p src/tsconfig.lib.json",
-    "postbuild": "rimraf -- src/inline-template",
-    "prepublish": "npm run build"
```

Comment:
Unfortunately I can confirm we want to remove the `prepublish` script, because it was executed by `npm install`.

If a built version of ng2-materialize exists, the source will not be used and it becomes an issue in development.